BookPlayer/Player/PlayerViewController.swift

```diff
@@ -110,17 +110,19 @@ class PlayerViewController: UIViewController, UIGestureRecognizerDelegate {
     }
 
     override func viewDidAppear(_ animated: Bool) {
-        if UserDefaults.standard.bool(forKey: Constants.UserDefaults.autolockDisabled.rawValue) {
-            UIApplication.shared.isIdleTimerDisabled = true
-        }
+        UIDevice.current.isBatteryMonitoringEnabled = true
+        NotificationCenter.default.addObserver(self, selector: #selector(self.onDeviceBatteryStateDidChange), name: UIDevice.batteryStateDidChangeNotification, object: nil)
```

Comment:
We can put this in `updateAutolock` and use the `UIDevice.current.isBatteryMonitoringEnabled` flag to know if the observer should be added